x-pack/plugins/watcher/__jest__/client_integration/helpers/watch_edit.helpers.ts

```diff
@@ -8,7 +8,7 @@
 import { registerTestBed, TestBed, AsyncTestBedConfig } from '@kbn/test-jest-helpers';
 import { HttpSetup } from '@kbn/core/public';
 
-import { WatchEdit } from '../../../public/application/sections/watch_edit/components/watch_edit';
+import { WatchEditPage } from '../../../public/application/sections/watch_edit_page';
```

Comment:
What do you think about updating the test helper file names as well to match the component names, e.g., `watch_edit_page.helpers.ts`?

Reply:
Good catch! Will fix.